src/components/Comments/WriteComment.js

```diff
@@ -33,8 +35,8 @@ const WriteComment = ({
   const [isAnonimity, setIsAnonimity] = React.useState(false);
   const isCommentEnabled = value.length > 0;
   const [loadingUser, setLoadingUser] = React.useState(false);
-  const isDisableSubmit = !isCommentEnabled || loadingPost;
   const [anonimityData, setAnoimityData] = React.useState({});
+  const storageKey = 'isAnonymByDefault';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `isDisableSubmit` and the introduction of `storageKey` variable are good changes. The `storageKey` will be used for storing user preferences related to anonymity, which improves the user experience by remembering their last choice.



<!-- This is an auto-generated comment by OSS CodeRabbit -->